client/src/main/java/org/schlunzis/kurtama/client/fx/controller/CreateServerController.java

```diff
@@ -0,0 +1,157 @@
+package org.schlunzis.kurtama.client.fx.controller;
+
+import javafx.application.Platform;
+import javafx.collections.FXCollections;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.stage.DirectoryChooser;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.schlunzis.kurtama.client.events.ClientClosingEvent;
+import org.schlunzis.kurtama.client.fx.view.StatusView;
+import org.schlunzis.kurtama.client.server.*;
+import org.schlunzis.kurtama.client.settings.IUserSettings;
+import org.schlunzis.kurtama.client.settings.Setting;
+import org.schlunzis.kurtama.client.util.I18n;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.util.Arrays;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class CreateServerController {
+
+    private final ServerFactory serverFactory;
+    private final LogSink logSink;
+    private final IUserSettings userSettings;
+    private final I18n i18n;
+    private Server server;
+
+    @FXML
+    private ComboBox<ServerType> typeSelector;
+    @FXML
+    private StatusView requirementsStatusView;
+    @FXML
+    private TextField portField;
+    @FXML
+    private TextField pathField;
+    @FXML
+    private Button selectFolderButton;
+    @FXML
+    private StatusView serverStatusView;
+    @FXML
+    private Button stopRunButton;
+    @FXML
+    private TextArea logArea;
+
+    @FXML
+    private void initialize() {
+        requirementsStatusView.setI18n(i18n);
+        serverStatusView.setI18n(i18n);
+        pathField.setText(userSettings.getString(Setting.SERVER_PATH));
+        portField.setText(String.valueOf(userSettings.getInt(Setting.PORT)));
```

Comment:
```suggestion
        portField.setText(Integer.toString(userSettings.getInt(Setting.PORT)));
```

I don't say, this change is needed. I simply wanted to show a different way.